app/src/main/java/org/fossasia/openevent/general/about/AboutEventFragment.kt

```diff
@@ -75,26 +73,23 @@ class AboutEventFragment : Fragment(), AppBarLayout.OnOffsetChangedListener {
         rootView.aboutEventContent.text = event.description?.stripHtml()
         val startsAt = EventUtils.getEventDateTime(event.startsAt, event.timezone)
         val endsAt = EventUtils.getEventDateTime(event.endsAt, event.timezone)
+
         rootView.aboutEventDetails.text = EventUtils.getFormattedDateTimeRangeBulleted(startsAt, endsAt)
         rootView.eventName.text = event.name
-    }
-
-    override fun onOffsetChanged(appBarLayout: AppBarLayout, verticalOffset: Int) {
-        val maxScroll = appBarLayout.totalScrollRange
-        val percentage = Math.abs(verticalOffset).toFloat() / maxScroll.toFloat()
+        Picasso.get()
+            .load(event.originalImageUrl)
+            .into(rootView.aboutEventImage)
```

Comment:
I think adding a placeholder might be a good idea

Reply:
yeah, it's a good idea, I will add it